capstone/src/main/java/com/google/univiz/scorecard/SuggestionResponse.java

```diff
@@ -1,4 +1,14 @@
 package com.google.univiz.scorecard;
 
-/** SuggestionResponse represents the list of college data specific to the Search feature */
-public class SuggestionResponse {}
+import com.google.auto.value.AutoValue;
+import com.google.gson.annotations.SerializedName;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+import java.util.List;
+
+/** SuggestionResponse represents the list of SuggestionData specific for the Search feature */
+@GenerateTypeAdapter
+@AutoValue
+public abstract class SuggestionResponse {
```

Comment:
does it need to be public?

Reply:
yes, otherwise I can't use it in the SuggestionDataApi package